ui_framework/src/components/form/form_row/form_row.js

```diff
@@ -110,4 +117,9 @@ KuiFormRow.propTypes = {
   isInvalid: PropTypes.bool,
   error: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
   helpText: PropTypes.string,
+  addEmptyLabelSpace: PropTypes.bool,
+};
+
+KuiFormRow.defaultProps = {
+  addEmptyLabelSpace: false,
```

Comment:
Small nit: can we change this to `hasEmptyLabelSpace`, to reflect the boolean nature of this prop?